docs/adr/template.md

```diff
@@ -9,39 +9,90 @@
   superseded by [ADR-xxx](./xxx.md)
   -->
 - **Deciders**: <!-- @github-user -->
-<!-- - Technical Story: [description | ticket/issue URL] -->
 
-## Context and Problem Statement
+## Context and problem statement
 
 <!--
 Describe the context and problem statement, e.g., in free form using two to
 three sentences. You may want to articulate the problem in form of a question.
 -->
 
 <!--
-## Decision Drivers
+### Issues with existing set-up
 
-- [driver 1, e.g., a force, facing concern, …]
-- [driver 2, e.g., a force, facing concern, …]
+Summary of problems with the existing design, possibly with some snippets.
 -->
 
-## Considered Options
+<!--
+### Technical story
+
+If available, list and link related issues, PRs, or ADRs that led up to this
+ADR:
+
+- [#000](https://github.com/ComPWA/expertsystem/issues/000)
+- [#000](https://github.com/ComPWA/expertsystem/issues/000)
+-->
+
+### Solution requirements
+
+1. <!-- Requirement 1 -->
+2. <!-- Requirement 2 -->
```

Comment:
I like the word decision drivers, as its maybe not always possible to define strict requirements. And decision drivers is generally just saying which aspects influence your decision making.

Maybe we can do something like this:

```md
## Decision drivers

### Solution requirements

1. <!-- Requirement 1 -->
￼2. <!-- Requirement 2 -->
```

Reply:
371aca1